docs/database-functionalities/triggers.md

```diff
@@ -0,0 +1,295 @@
+---
+id: triggers
+title: Triggers
+sidebar_label: Triggers
+---
+
+This article is a part of a series intended to showcase Memgraph's features
+and bring the user up to speed on developing with Memgraph.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [database functionalities section](./database-functionalities.md).
+
+## Introduction
+
+Memgraph supports running openCypher statements after a certain event happens during transaction execution, i.e. triggers.
+
+As user, you can create triggers, delete them, and print them out.
+All the triggers are persisted on the disk, so no information is lost between the runs.
+
+## Creating a trigger
+
+To create a new trigger, a query of the following format should be used:
+
+```plaintext
+CREATE TRIGGER trigger_name ( ON ( () | --> ) CREATE | UPDATE | DELETE )
+( BEFORE | AFTER ) COMMIT
+EXECUTE openCypherStatements
+```
+
+At first, the query may seem complex, so let's talk about the different parts of it independently.
+
+### Trigger name
+Each created trigger should be defined with a name. The name needs to be unique globally.
+E.g. two triggers for different event still can't have the same name.
+
+### Event type
+Optionally, users can define for which event should the trigger execute its statements.
+The event type is defined using the following part:
+
+```plaintext
+ON ( () | --> ) CREATE | UPDATE | DELETE
+```
+
+There are three main event types:
+  - CREATE
+  - UPDATE
+  - DELETE
+
+For each of the event types, user can denote if he wants to execute trigger's statements just on the events
+that happened on a vertex or just on an edge. Vertex is denoted using `()` and edges using `-->`.
+
+Few examples would be:
+`ON CREATE` - trigger the statements only if an object (vertex and/or edge) was created during the transaction execution.
+`ON () UPDATE` - trigger the statements only if a vertex was updated (e.g. property was set on it) during the transaction execution.
+`ON --> DELETE` - trigger the statements only if an edge was deleted during the transaction execution.
+
+Each event comes with certain information that can be used in the openCypher statements the trigger executes. The information is contained in the
+form of predefined vairables. (LINK TO THE PART ABOUT THE PREDEFINED VARIABLES)
+
+If no event type is specified, the trigger executes its statements every time, and all the predefined variables can be used.
+
+### Phase of executing statetements
+Trigger can execute its statements at specified phase, before or after comitting the transaction that triggered it.
```

Comment:
I would also change `Trigger` -> `A trigger`, because that sounds more natural for me. Though grammarly agrees with me, I am still not sure. So feel free to keep this as is if you think this is better.

The same applies for below at line 68.